Solutions/Marain.UserNotifications.Abstractions/Marain/NotificationTemplates/CommunicationTemplates/EmailTemplate.cs

```diff
@@ -0,0 +1,43 @@
+﻿// <copyright file="EmailTemplate.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Marain.NotificationTemplates.CommunicationTemplates
+{
+    /// <summary>
+    /// Email Object.
+    /// </summary>
+    public class EmailTemplate : ICommunicationTemplate
+    {
+        /// <summary>
+        /// The content type that will be used when serializing/deserializing.
+        /// </summary>
+        public const string RegisteredContentType = "application/vnd.marain.usernotifications.notificationtemplate.emailtemplate.v1";
+
+        /// <inheritdoc/>
+        public string? NotificationType { get; set; }
+
+        /// <inheritdoc/>
+        public string? ETag { get; set; }
+
+        /// <summary>
+        /// Gets or sets the body of the email.
+        /// </summary>
+        public string? Body { get; set; }
```

Comment:
Hi Ian, These templates are currently rough (placeholders) and not used as we have planned to work on these in the next month. These were just added with the WebPush Template so that we have multiple scenarios to code against.

Reply:
OK.